app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -213,9 +207,14 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
         registerForActivityResult.launch(intent)
     }
 
-    private fun claimStatusCards(successData: HomeQuery.Data): HomeModel.ClaimStatus = HomeModel.ClaimStatus(
-        successData.claimStatusCards.map(ClaimStatusCardData::fromClaimStatusCardsQuery)
-    )
+    private fun MutableList<HomeModel>.addClaimStatusCardsIfApplicable(successData: HomeQuery.Data) {
+        if (featureManager.isFeatureEnabled(Feature.CLAIMS_STATUS).not()) return
+        NonEmptyList.fromList(successData.claimStatusCards)
+            .map { claimStatusCardsQuery -> claimStatusCardsQuery.map(ClaimStatusCardData::fromClaimStatusCardsQuery) }
+            .tap { claimStatusCardDataList ->
```

Comment:
`tap` function signature looks like this:
```
/**
* The given function is applied as a fire and forget effect
* if this is a `some`.
* When applied the result is ignored and the original
* Some value is returned
*
* Example:
* ```
* Some(12).tap { println("flower") } // Result: prints "flower" and returns: Some(12)
* none<Int>().tap { println("flower") }  // Result: does not print anything and returns: None
* ```
*/
public inline fun tap(f: (A) -> Unit): Option<A> =
when (this) {
    is None -> this
    is Some -> {
        f(this.value)
        this
    }
}
```

Basically short for having to do a when on the result and do nothing on `None` and what you want to do on `Some`